client/components/fields/agendas.tsx

```diff
@@ -1,31 +1,35 @@
-import React, {Fragment} from 'react';
-import PropTypes from 'prop-types';
+import React, {Fragment, FunctionComponent} from 'react';
 import classNames from 'classnames';
-import {get} from 'lodash';
 
 import {AgendaNameList} from '../Agendas';
+import {superdeskApi} from '../../superdeskApi';
 
-export const agendas = ({item, fieldsProps, noGrow}) => (
-    <Fragment>
-        <span className="sd-list-item__text-label">Agenda:</span>
-        <span
-            className={classNames(
-                'sd-overflow-ellipsis',
-                'sd-list-item__text-strong',
-                {
-                    'sd-list-item--element-grow': !noGrow,
-                }
-            )}
-        >
-            <AgendaNameList agendas={get(fieldsProps, 'agendas.agendas')} />
-        </span>
-    </Fragment>
-);
+interface IProps {
+    fieldsProps: {
+        agendas: {
+            agendas: any;
```

Comment:
I believe this might be [IAgenda](https://github.com/superdesk/superdesk-planning/blob/develop/client/interfaces.ts#L372)